pages/api/webhooks/order-fully-paid.ts

```diff
@@ -0,0 +1,43 @@
+import { NextApiRequest, NextApiResponse, NextApiHandler } from "next";
+import { SALEOR_DOMAIN_HEADER } from "../../../constants";
+import Klaviyo from "../../../lib/klaviyo";
+
+const expectedEvent = "order_fully_paid";
+
+const handler: NextApiHandler = async (
+  request: NextApiRequest,
+  response: NextApiResponse
+) => {
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
+  if (!saleorDomain) {
+    response
+      .status(400)
+      .json({ success: false, message: "Missing saleor domain token." });
+    return;
+  }
+
+  const receivedEvent = request.headers["saleor-event"]?.toString();
+
+  if (receivedEvent !== expectedEvent) {
+    return response
+      .status(400)
+      .json({ success: false, message: "Invalid event" });
+  }
+
+  const context = request.body;
+  const userEmail = context[0]?.user_email;
```

Comment:
I don't believe recipient's email for every event will be set in this particular leaf of payload. I would recommend to test it using subscriptions' definitions from https://github.com/saleor/saleor-app-klaviyo/pull/4, maybe we should extend GraphQL definitions there?